sites/example-project/src/pages/+layout.js

```diff
@@ -46,7 +46,9 @@ const database_initialization = profile(loadDB);
 /** @type {(...params: Parameters<import("./$types").LayoutLoad>) => Promise<App.PageData["data"]>} */
 async function getPrerenderedQueries({ data: { routeHash, paramsHash }, fetch }) {
 	// get every query that's run in the component
-	const res = await fetch(`/api/${routeHash}/${paramsHash}/all-queries.json`);
+	const res = await fetch(`/api/${routeHash}/${paramsHash}/all-queries.json`).catch(() => ({
+		ok: false
+	}));
```

Comment:
https://stackoverflow.com/questions/4500741/suppress-chrome-failed-to-load-resource-messages-in-console/75848002#75848002